resources/lrsql/config/oidc_authority.json.template

```diff
@@ -0,0 +1,27 @@
+{# for template syntax see: https://github.com/yogthos/Selmer#templates
+
+   Use any variable present in token claims
+   https://datatracker.ietf.org/doc/html/rfc9068
+
+   Special Variables:
+
+   * lrsql/resolved-client-id - Replacement for aud that is always a string.
```

Comment:
Maybe document here that `iss` and `sub` are fields that will come from the JWT.

Reply:
That link leads to a master list of everything that could (possibly) be in the claims with definitions, isn't that enough?

Reply:
oh, that's not actually the doc I thought I linked there

Reply:
see https://github.com/yetanalytics/lrsql/pull/205/commits/39e22ef25147879a888332da2e6238cdf8c3696a